sirepo/sim_api/jupyterhublogin.py

```diff
@@ -29,6 +29,7 @@
 class API(sirepo.quest.API):
     @sirepo.quest.Spec("require_user", sim_type=f"SimType const={_SIM_TYPE}")
     async def api_checkAuthJupyterHub(self):
+        # check that user has role
```

Comment:
this will be going away eventually https://github.com/radiasoft/sirepo/issues/7026. I added a more detailed comment. [c726701](https://github.com/radiasoft/sirepo/pull/7027/commits/c72670133ae55943f058d0447b6b4ff060741d6d)